src/mt_protocol.cpp

```diff
@@ -119,10 +125,6 @@ bool mt_send_text(const char * text, uint32_t dest, uint8_t channel_index) {
   toRadio.which_payload_variant = meshtastic_ToRadio_packet_tag;
   toRadio.packet = meshPacket;
   
-  Serial.print("Sending text message '");
```

Comment:
Does this need to be removed?

Reply:
No it shouldnt. I returned it back with IF to show it only during debugging.